interactions_test.go

```diff
@@ -140,7 +140,8 @@ const (
           "some_datetime": {
             "value": {
               "type": "datetimepicker",
-              "selected_date_time": null
+              "selected_date_time": null,
+							"timezone": "Europe/Berlin"
```

Comment:
This is in the wrong type. `timepicker` has a `timezone`, but `datepicker` and `datetimepicker` don't

Reply:
oopsie - thanks for spotting